muikku-persistence/src/main/java/fi/otavanopisto/muikku/dao/users/UserEntityFileDAO.java

```diff
@@ -42,6 +42,10 @@ public UserEntityFile updateData(UserEntityFile userEntityFile, String identifie
     getEntityManager().persist(userEntityFile);
     return userEntityFile;
   }
+  
+  public void delete(UserEntityFile userEntityFile) {
+    getEntityManager().remove(userEntityFile);
```

Comment:
Replace method content with...

super.delete(userEntityFile);

...as the underlying code will not only delete the entity but also fire an event about it.